src/net.cpp

```diff
@@ -651,6 +651,10 @@ void CNode::copyStats(CNodeStats &stats)
     stats.cleanSubVer = cleanSubVer;
     stats.fInbound = fInbound;
     stats.nStartingHeight = nStartingHeight;
+    {
+        LOCK(cs_filter);
+        stats.fRelayTxes = fRelayTxes;
+    }
```

Comment:
Bug: this should be changing the third line of this function, not adding a new section.